### PR TITLE
Made Body::containsPoint() consistent in surface points inclusion. (melodic)

### PR DESCRIPTION
**Pull request by peci1**
Retargeted #90 to melodic-devel.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,8 @@ Supported shapes:
 
 Note: Bodies for meshes compute the convex hull of those meshes in order to provide the point containment / ray intersection routines.
 
+Note: There was a recent change in point containment computations - now all bodies are expected to also contain their surface points. [The old behavior was inconsistent](https://github.com/ros-planning/geometric_shapes/issues/91). Rounding errors may still yield surprising results, though.
```
Comment (rhaschke) [inline README.md]: This should be moved into MIGRATION.md.

**Review by rhaschke — CHANGES_REQUESTED** [review]
Thanks for retargeting this to Melodic. Could please squash the last two commits (fix formatting).

**Comment by peci1** [comment]
Squashed and created MIGRATION.md (there was none before).

**Comment by rhaschke** [comment]
> Squashed and created MIGRATION.md (there was none before).

Ah, sorry, we have a MIGRATION.md only for the main MoveIt repository.
I'm not sure, it's worth to mention these changes in README.md or a new MIGRATION.md.
Usually such (minor) changes are reported in the ChangeLog only. What do you think?

**Comment by peci1** [comment]
Well, the newly created MIGRATION.md could also contain information about the merge with shape_tools... I think it might come handy...

**Pull request merged**